angr_platforms/msp430/instrs_msp430.py

```diff
@@ -33,29 +33,35 @@ def bits_to_signed_int(s):
     return Bits(bin=s).int
 
 class MSP430Instruction(Instruction):
+    commit_func = None
```

Comment:
Could you move this to an `__init__`, just to make it clear that it is instance specific?